Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -286,13 +294,9 @@ class TabLocationView: UIView {
 
     fileprivate func updateTextWithURL() {
         if let host = url?.host, AppConstants.MOZ_PUNYCODE {
-            urlTextField.text = url?.absoluteString.replacingOccurrences(of: host, with: host.asciiHostToUTF8())
+            urlTextField.text = url?.absoluteString.replacingOccurrences(of: host, with: host.asciiHostToUTF8()).etldValue(parser: parser) ?? ""
```

Comment:
Are we sure we want to show an empty `""` String in case of failure? That could confuse users. Any idea of a better fallback? cc @jhreis

Reply:
The nil coalescing will only happen when Url itself is nil. In fact urltextfield.text is an optional property, so removing nil coalescing(Had it in place for some other reason and forgot to remove it.)